### DISCUSSION TITLE
Recommended way to float plugin code-block elements like a leaflet map

### DISCUSSION THREAD

**Discussion by Jnosh** [SlRvb/Obsidian--ITS-Theme#177] [Q&A] [answered]
I sometimes want to float plugin elements defined through code-blocks like e.g. a leaflet map on the right or left like ITS allows with callouts or embedded images.

Is there a recommended way to do the same with such plugin-defined content or is it too dependent on the plugin CSS styling and best to just try what works and add custom CSS snippets if needed?

Something like the following seems to work with leaflet for example but I'm not sure if there is a better way to do this that I should be using.

````
> [!|right clean no-title]
> ```leaflet
> id: map
> height: 300px
> width: 300px
> ...
> ```
````

**Comment by SlRvb** [comment, marked as answer]
No real recommended way, that's the way that I would do it with callouts for floating plugin elements. I can't imagine plugins wouldn't work with that method unless they're doing something really weird.
